app/utils/translation.py

```diff
@@ -1,13 +1,17 @@
 import json
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
 
 def load_translation(language):
     primary_language = language.split(",")[0].split(";")[0].split("-")[0]
-    print(f"Primary: {primary_language}")
+    logger.debug(f"Primary: {primary_language}")
```

Comment:
I think here the info level is more appropriate, I would replace logger.debug() with logger.info()

Reply:
Ah. way better